File: lib/speech_result.dart
class SpeechResult {
  String result;
  bool isPartial;

  SpeechResult({this.result, this.isPartial});

  SpeechResult.fromJson(Map<String, dynamic> json) {
    result = json['result'];
    isPartial = json['isPartial'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['result'] = this.result;
    data['isPartial'] = this.isPartial;
    return data;
  }

  @override
  String toString() {
    return 'SpeechResult{result: $result, isPartial: $isPartial}';
  }
}

File: android/src/main/kotlin/com/umair/background_stt/SpeechResult.kt
package com.example.background_tts_stt

data class SpeechResult(var result: String = "", var isPartial: Boolean) {

    override fun toString(): String {
        return "{" +
                "  \"result\": \"$result\"," +
                "  \"isPartial\": $isPartial" +
                "}"
    }
}